docs/pages/reference/resource-health-checks.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Teleport Resource Health Checks
+description: This reference explains what Teleport resource health checks are, how they work, how to configure them, and how to view resource health status.
+---
+
+Teleport resource health checks are used to monitor connectivity between your Teleport cluster and resources enrolled in your cluster.
+
+Currently only TCP health checks are supported and only for enrolled databases.
+
+## How it works
+
+Teleport's `health_check_config` API resource determines which resources will have health checks enabled and what settings will be used for those checks.
+Starting in Teleport 18, a preset default `health_check_config` will be created that enables TCP health checks for all enrolled databases.
+When health checks are enabled for a resource, Teleport services that proxy access to that resource will perform regular health checks against that resource's endpoint(s).
+The interval between health checks and timeout duration of each health check is configurable.
+
+Every supported resource has a health status: "healthy", "unhealthy", or "unknown".
+Resources without health checks enabled will have "unknown" health status.
+Additionally, Teleport services running on versions that do not support health checks will not report health status for resources they proxy.
+
+Each health checked resource will have an "unknown" health status until its initial health check result is available.
+The initial health check for a resource will determine its health status, healthy or unhealthy, regardless of configured healthy or unhealthy threshold.
+After the initial health check, health status will only change after consecutive passing or failing health checks reach the configured healthy or unhealthy threshold.
+
+<Admonition type="tip">
+Teleport proxies will use service-to-resource health status to make connection routing decisions when multiple services proxy access to the same enrolled resource.
+For example, if multiple Teleport database services proxy access to the same database, but only one of them can reach the database endpoint, then only one database service will report a "healthy" status.
+Teleport proxies will route all database user connections to that database through the service that reports healthy status instead of the services reporting unhealthy status.
+Proxies will prioritize routes to services with health status in the order: healthy > unknown > unhealthy.
+</Admonition>
+
+## Configuration
+
+Health check configuration specifies how Teleport should verify the health of connectivity between Teleport services and enrolled resources.
+It includes settings for the health check interval duration, timeout duration, healthy and unhealthy thresholds, and label matchers that determine which resources should be health checked.
+
+`tctl [create | get | edit | rm]` supports `health_check_config` resources (and its alias: `hcc`).
+For example, you can view the default health check configuration using `tctl get`:
```

Comment:
Could we provide one or more examples that actually uses the settings in the table below and for db_label_expressions.

Should we also mention you can have multiple configs?